### PR TITLE
Add tio.run and ideone.com to list of online repls

### PR DESCRIPTION
**Pull request by MasterDuke17**

### CODE REVIEW

File: source/resources/index.html [hunk 1]
```diff
@@ -112,10 +112,12 @@ <h3>Social Media</h3>
...
             <h3>Practice Online</h3>
-          	<ul class="shy-list">
-              <li><a href="http://exercism.io/languages/perl6">Perl&nbsp;6 track on exercism.io</a></li>
-    	        <li><a href="https://glot.io/new/perl6">Online Perl&nbsp;6 REPL</a></li>
-            </ul>
+              <ul class="shy-list">
+                  <li><a href="http://exercism.io/languages/perl6">Perl&nbsp;6 track on exercism.io</a></li>
+       	          <li><a href="https://glot.io/new/perl6">Online Perl&nbsp;6 REPL</a></li>
+                  <li><a href="https://tio.run/#perl6">Online Perl&nbsp;6 REPL</a></li>
+                  <li><a href="https://ideone.com/">Online Perl&nbsp;6 REPL</a></li>
```
Comment (zoffixznet) [inline source/resources/index.html]: Isn't it weird to have three repeated links that say "Online Perl 6 REPL"? Could've used the name of the REPL or something.

**Comment by AlexDaniel** [comment]
FWIW there's also https://f.perlbot.pl/#perl6

**Comment by zoffixznet** [comment]
> FWIW there's also https://f.perlbot.pl/#perl6

It's not very usable. It spams IRC channels (with no apparent means to opt out) and it doesn't let you REPL anything: once you submit, even if you hit "Fork and Edit" you're thrown into Perl 5's evaluator.

**Comment by bduggan** [comment]
It's also possible to launch a Perl 6 notebook on mybinder.org by following a link like [this](https://mybinder.org/v2/gh/bduggan/p6-jupyter-kernel/master?filepath=eg%2Fhello-world.ipynb)

**Pull request merged**